dspace-server-webapp/src/test/java/org/dspace/app/rest/StatisticsRestRepositoryIT.java

```diff
@@ -132,26 +140,27 @@ public void usagereports_withoutId_NotImplementedException() throws Exception {
 
     @Test
     public void usagereports_notProperUUIDAndReportId_Exception() throws Exception {
-        getClient().perform(get("/api/statistics/usagereports/notProperUUIDAndReportId"))
+        getClient(adminToken).perform(get("/api/statistics/usagereports/notProperUUIDAndReportId"))
                    .andExpect(status().is(HttpStatus.BAD_REQUEST.value()));
     }
 
     @Test
     public void usagereports_nonValidUUIDpart_Exception() throws Exception {
-        getClient().perform(get("/api/statistics/usagereports/notAnUUID" + "_" + TOTAL_VISITS_REPORT_ID))
+        getClient(adminToken).perform(get("/api/statistics/usagereports/notAnUUID" + "_" + TOTAL_VISITS_REPORT_ID))
```

Comment:
For these Exception methods, can we add an test for anonymous users back in?  It looks like you removed/replaced it and are now only proving that exceptions work properly for Administrative users.  Since the default setting is that anonymous users can access stats, we need to prove that exception handling works properly for anonymous users as well.